backend/api/serializers/computetask.py

```diff
@@ -148,8 +152,12 @@ def get_asset(self, task_output_asset):
             model = Model.objects.get(key=task_output_asset.asset_key)
             return ModelSerializer(context=self.context, instance=model).data
         elif task_output_asset.asset_kind == FunctionOutput.Kind.ASSET_PERFORMANCE:
-            task_key, metric_key = task_output_asset.asset_key.split("|")
-            performance = Performance.objects.get(compute_task__key=task_key, metric__key=metric_key)
+            compute_task_key, metric_key, identifier = task_output_asset.asset_key.split("|")
```

Comment:
Does it still make sense to have both a `get_value` and a `get_asset` or is one of the two soon to be deprecated? 
Again, not the point of this PR, but for my understanding 🙂

Reply:
Summoning @guilhem-barthes as he already spend some time on the question